packages/server/graphql/types/GitLabSearchQuery.ts

```diff
@@ -0,0 +1,24 @@
+import {GraphQLID, GraphQLNonNull, GraphQLObjectType, GraphQLString} from 'graphql'
+import {GQLContext} from '../graphql'
+import GraphQLISO8601Type from './GraphQLISO8601Type'
+
+const GitLabSearchQuery = new GraphQLObjectType<any, GQLContext>({
+  name: 'GitLabSearchQuery',
+  description: 'A GitLab search query including the search query and the project filters',
+  fields: () => ({
+    id: {
+      type: new GraphQLNonNull(GraphQLID),
+      description: 'shortid'
+    },
+    queryString: {
```

Comment:
-1 i think you need to add `projectIdFilters` unless you're gonna concat those into the query string?

Reply:
As we're just doing the search in this PR, I didn't include it here, but it does exist in the subsequent PR when we introduce filtering: https://github.com/ParabolInc/parabol/pull/6323